catalog/admin/admin-poller-service/src/main/java/org/codice/ddf/catalog/admin/poller/AdminPollerServiceBean.java

```diff
@@ -215,6 +251,109 @@ public int compare(Map<String, Object> o1, Map<String, Object> o2) {
         return metatypes;
     }
 
+    @Override
+    public List<Serializable> updatePublications(String source, List<String> destinations)
+            throws UnsupportedQueryException, SourceUnavailableException, FederationException {
+        List<Serializable> updatedPublishedLocations = new ArrayList<>();
+
+        if (destinations != null) {
```

Comment:
Might be a little cleaner to just return the empty list if destinations is null or empty